core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -1344,7 +1350,7 @@ wasm_cluster_is_thread_terminated(WASMExecEnv *exec_env)
 {
     os_mutex_lock(&exec_env->wait_lock);
     bool is_thread_terminated =
-        (exec_env->suspend_flags.flags & 0x01) ? true : false;
+        (WASM_SUSPEND_FLAGS_GET(exec_env->suspend_flags) & 0x01) ? true : false;
```

Comment:
Had better change `0x0` to `WASM_SUSPEND_FLAG_TERMINATE`

Reply:
done